docs/helm-chart/manage-dags-files.rst

```diff
@@ -177,6 +177,38 @@ In this example, you will create a yaml file called ``override-values.yaml`` to
 
 Don't forget to copy in your private key base64 string.
 
+Using SSH Key from Hashicorp Vault instead of Secret
+----------------------------------------------------
```

Comment:
Being injected in the middle of this existing subsection is confusing I think - the note about using knownhosts is not in the original subsection any longer.

Reply:
Where is the proper place to put it?